cheroot/test/test_server.py

```diff
@@ -439,3 +441,52 @@ def many_open_sockets(request, resource_limit):
         # Close our open resources
         for test_socket in test_sockets:
             test_socket.close()
+
+
+@pytest.mark.parametrize(
+    ('minthreads', 'maxthreads', 'error'),
+    (
+        (-1, -1, ValueError),
+        (-1, 0, ValueError),
+        (-1, 1, ValueError),
+        (-1, 2, ValueError),
+        (0, -1, ValueError),
+        (0, 0, ValueError),
+        (0, 1, ValueError),
+        (0, 2, ValueError),
+        (1, -1, None),
+        (1, 0, ValueError),
+        (1, 1, None),
+        (1, 2, None),
+        (2, -1, None),
+        (2, 0, ValueError),
+        (2, 1, ValueError),
+        (2, 2, None),
+    ),
+)
+def test_threadpool_threadnum_validation(minthreads, maxthreads, error):
+    kwargs = dict(
+        server=None,
+        min=minthreads,
+        max=maxthreads,
+    )
+    if error is None:
+        ThreadPool(**kwargs)
+    else:
+        with pytest.raises(error):
+            ThreadPool(**kwargs)
+
+
+def test_threadpool_multistart_validation(monkeypatch):
```

Comment:
done!

Reply:
FYI the linters check for PEP 257.

Reply:
I think I fixed it now with the closing quote on a new line

Reply:
Adding the "Raises:" in the same format as I saw elsewhere in the code didn't seem to placate the linter for some reason but all it;s other complaints about the docstrings have been resolved.

Reply:
I don't see the related error in CI so you probably shouldn't worry about that. `Raises:` is so-called “napoleon” format for docstring metadata but the `darglint` plugin enforces the Sphinx-native style which needs `:raises:`.
FWIW it's not enabled everywhere because of the inconsistency in use. Those napoleon-style definitions should be converted to the native sphinx style over time.